daemon/mgr/container_exec.go

```diff
@@ -97,6 +98,23 @@ func (mgr *ContainerManager) StartExec(ctx context.Context, execid string, cfg *
 		},
 	}
 
+	if execConfig.Privileged {
+		capList := caps.GetAllCapabilities()
+		process.Capabilities = &specs.LinuxCapabilities{
+			Effective:   capList,
+			Bounding:    capList,
+			Permitted:   capList,
+			Inheritable: capList,
+		}
+	} else {
```

Comment:
just combine else and if to be `else if {` in this case? @Ace-Tang

Reply:
But no `else if` here

Reply:
Since in the `else`, there is only one `if`. So I am wondering if we could make it 
```
else if spec, err := mgr.spec(c); err == nil {
    process.Capabilities = spec.Process.Capabilities
}

```